app/src/Data/Card.elm

```diff
@@ -12,6 +12,7 @@ type alias Card =
     , votes : Int
     , totalVotes : Int
     , contents : List Content
+    , editing: Bool
```

Comment:
This might be better placed at the Content level? https://github.com/hawx/retro/pull/54#issuecomment-335021597